src/libs/antares/study/parts/short-term-storage/properties.h

```diff
@@ -52,7 +53,7 @@ unsigned int groupIndex(Group group);
 class Properties
 {
 public:
-    bool validate();
+    bool validate(Antares::Data::SimplexOptimization simplex);
```

Comment:
About `bool validate(Antares::Data::SimplexOptimization simplex);` : 
As **simplex** can only take 2 values (Day or Week), maybe we could have instead : 
`bool validate(bool isWeeklySimplex);`
This would avoid dependency from **parameters.h**.

Reply:
Was done previously with an int by we switched to the enum